spec/controllers/works_controller_spec.rb

```diff
@@ -195,13 +195,13 @@ def call_with_params(params)
 
   describe "create" do
     before do
-      @user = FactoryGirl.create(:user)
+      @user = create(:user)
```

Comment:
Just succinctness. FactoryGirl::Syntax::Methods is loaded into the Rspec config in the spec_helper so the methods are available directly, and I noticed that we have both syntaxes throughout our tests (with and without `FactoryGirl.`). Since there appears to be no ambiguity using `create` vs `FactoryGirl.create`, I thought I'd standardise this particular spec file on the shorter version to remove visual clutter.